### ISSUE TITLE
[Help] OBS could not play the rstp source

### ISSUE THREAD

**Issue by bao3** [exelix11/SysDVR#60]
**Where do you get stuck ?**
https://github.com/exelix11/SysDVR/wiki/Live-streaming-with-OBS

OBS could not play the rtsp source , but mpv could.

**What's the error message you get ?**

I read the OBS log :

```
04:42:59.004: [Media Source '媒体源']: settings:
04:42:59.004: 	input:                   rtsp://127.0.0.1:6666
04:42:59.004: 	input_format:            
04:42:59.004: 	speed:                   100
04:42:59.004: 	is_looping:              no
04:42:59.004: 	is_hw_decoding:          no
04:42:59.004: 	is_clear_on_media_end:   yes
04:42:59.004: 	restart_on_activate:     yes
04:42:59.004: 	close_when_inactive:     yes
04:42:59.006: Switched to scene '场景'
04:42:59.006: ------------------------------------------------
04:42:59.006: Loaded scenes:
04:42:59.006: - scene '场景':
04:42:59.006:     - source: '媒体源' (ffmpeg_source)
04:42:59.006: ------------------------------------------------
04:42:59.437: MP: Failed to open media: 'rtsp://127.0.0.1:6666'
04:42:59.769: adding 69 milliseconds of audio buffering, total audio buffering is now 69 milliseconds (source: 麦克风/Aux)
04:42:59.769: 
04:46:44.362: Failed to read log file /Users/baoshan_wang/Library/Application Support/obs-studio/crashes/

```

And I tried to play the source from VLC Play , but it cannt play . If I  use mpv ,then I could play the rtsp://127.0.0.1:6666/ .

**Setup information**
 
On macOS , I run :
`dotnet SysDVR-Client.dll bridge 172.16.11.204 ` 
**Additional context**
None

I just followed https://github.com/exelix11/SysDVR/wiki/Live-streaming-with-OBS

**Comment by exelix11** [comment]
Not sure, if mpv works your setup should be correct, only thing i can think of is that the log reads: 
> 04:42:59.004: 	input:                   rtsp://127.0.0.1:6666

while the correct url is `rtsp://127.0.0.1:6666/`, the trailing slash matters. So make sure to be using the full url it in both vlc and obs.

If that still doesn't work it could be related to network setup (?) really not sure as it shouldn't matter for localhost connections.

**Comment by tkgstrator** [comment]
SysDVR: 4.0.1
OBS: 25.0.8

It is same issue for me. mpv works well but obs did not display the capture image.

**Comment by exelix11** [comment]
> In my research, first we have to run SysDVR-Client at first before launch OBS.
Uhm well, obviously, or else what does it connect to ? Is the guide unclear ?

**Comment by tkgstrator** [comment]
> Is the guide unclear ?

I think it is better that document explains users have to run SysDVR-Client before OBS try to access RTSP.

**Comment by exelix11** [comment]
The page currently says
> Launch RTSP streaming like you usually do, if you use SysDVR-ClientGUI don't select an mpv path when launching.

And

> 127.0.0.1 If you're using SysDVR-Client for TCP Bridge or USB streaming.

Sounds clear to me, guess will think of a better wording

**Comment by tkgstrator** [comment]
Ah, I see.
Btw, is it hard to re-establish the connection automatically?

**Comment by exelix11** [comment]
I have no control on that as it depends on the player.
From my tests obs will try to connect again after a bit if it drops, mpv needs to be restarted.

**Comment by exelix11** [comment]
Closing this as with the just released 5.0 update it's recommended to use the built-in video player.
